docs/userGuide/syntax/questions.mbdf

```diff
@@ -1,130 +1,343 @@
+  </small>
+  </pic>
+
+  (**Question Body**)
+
+  <!-- Several hidden checkbox q-option components explained later -->
+
+  <!-- Hint slot -->
+  <template slot="hint">
+
+  Think out of the box! :fas-box:
+
+  Need another hint? <tooltip content="Two of the answers are correct!">Hover over me!</tooltip> :fas-mouse-pointer:
+  </template>
 </question>
 </variable>
-</include>
+<variable name="output">
+<question type="checkbox" header="Which of the following is true? (**Header**)" hint="Think out of the box! :fas-box:">
+  <!-- Header slot -->
+  <template slot="header">
 
-**You are able to omit the hint or answer `<div>` independently, and they will not render.**
+  Which of the following is correct? (**Header**)
 
-<include src="codeAndOutput.md" boilerplate >
-<variable name="highlightStyle">html</variable>
-<variable name="code">
-<question>
-  This question with an omitted hint.
-  <div slot="answer">42 <md>:star:</md></div>
-</question>
+  Challenge: Try to get all the answers on your first try! :star: :star:
+  </template>
 
-<question>
-  This question with an omitted answer.
-  <div slot="hint">The number of exponent bits in a 64-bit Floating Point. <md>:computer:</md></div>
-</question>
+  <pic src="{{baseUrl}}/images/math-question.jpg" alt="math question image" height="200" class="d-block mx-auto">
+
+  <small>Adapted from [Daily Mail](https://www.dailymail.co.uk/femail/article-4702868/Can-pass-intelligence-test.html)
+  </small>
+  </pic>
+
+  (**Question Body**)
+
+  <q-option correct reason="Multiply the numbers on the left together and add the leftmost number!">
+    96
+  </q-option>
+  <q-option reason="Under normal circumstances, this would be correct.">
+    19
+  </q-option>
+  <q-option correct reason="Simply add the running sum of the results as well!">
+    40
+  </q-option>
+  <q-option>
+    811
+  </q-option>
+
+  <!-- Hint slot -->
+  <template slot="hint">
 
-<question>
-  This question only has the question body.
+  Think out of the box! :fas-box:
+
+  Need another hint? <tooltip content="Two of the answers are correct!">Hover over me!</tooltip> :fas-mouse-pointer:
+  </template>
 </question>
 </variable>
 </include>
+</panel>
+
+<box type="tip" seamless class="mt-3">
 
-**Use the minimized Panel component to create and group questions inline.**
+Placing the question into the header is entirely optional. You may also wish to include the question directly in the question body, omitting the header entirely.
+</box>
+
+
+****Common `<question>` Options and Slots****
+Name | Type | Default | Description
+--- | --- | --- | ---
+header | `String` | `''` | The markup to insert into the question header. The header is omitted if this is not provided.
+header | Slot | empty | Functions similar to the `header` option, and overrides it if both are present.
+hint  | `String` | `''` | The content to display in the hint box.
+hint  | Slot | empty | Functions similar to the `hint` option, and overrides it if both are present.
```

Comment:
I think we should some other way to indicate which attributes support the alternative slot syntax, rather than add one row for each. e.g., use a header<sup>[s]</sup>

Reply:
hmm, I was thinking we should actually introduce slots upfront in the components section and not all the way below in `advanced tips and tricks`, since they are widely used in the examples. (don't think we should forward reference in user guides). How about cleaning and standardising this in that PR?

Reply:
No need for forward reference. e.g., we can make it a modal/popover. Define in one place and `<include>` everywhere needed. No matter where the reader encounters it first, the definition is just a hover/click away.

Reply:
Sounds good. I'll add it in a pr after the one fixing the rest of the comments here